src/test/java/tests.java

```diff
@@ -0,0 +1,135 @@
+import org.json.JSONException;
+import org.junit.Before;
+import org.junit.Test;
+import srver.PO;
+import srver.server;
+import srver.vulnerability;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static srver.server.readJsonFromUrl;
+
+
+/**
+ * Created by Ксения on 12.12.2016.
+ */
+
+public class tests {
+    public vulnerability vuln;
+
+
+
+
+    @Before
+    public void before() {
+        server s = new server();
+       /* vuln =new vulnerability("lastseen", "references", "description", "edition", "published",
+               "title", "type", "objectVersion",  "bulletinFamily",
+               "cvelist", "modified", "href", "id", "score");*/
+
+
+    }
+
+    @Test
+    public void testreadJsonFromUrl() throws IOException, JSONException {
+
+        assertNotNull(readJsonFromUrl("https://vulners.com/api/v3/search/lucene/?query=type:centos%20cvss.score:[8%20TO%2010]%20order:published"));
+
+    }
+
+    @Test
+    public void getNamePO() throws IOException, JSONException {
```

Comment:
&#x1F34E; **Issue**
Also, in tests you can just throw "Exception" instead of a myriad of different exceptions. (though, in actual code its better to throw specific exceptions and not combine them into one "Exception").